### PR TITLE
Added people random type and divided functions between different files

### PR DESCRIPTION
**Pull request by Maxi-Mega**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +18,15 @@
...
-    "moment-timezone": "^0.5.32",
-    "js-sha1": "^0.6.0",
     "ffmpeg": "0.0.4",
     "ffmpeg-static": "^4.2.7",
     "fluent-ffmpeg": "^2.1.2",
-    "ytdl-core": "^4.0.5",
-    "@discordjs/opus": "^0.3.3"
+    "js-sha1": "^0.6.0",
+    "moment-timezone": "^0.5.32",
+    "xpath-html": "^1.0.2",
```
Comment (zaosoula) [inline package.json]: This package doesn't seem to be used 
```suggestion
```

File: commands/random/index.js [hunk 2]
```diff
@@ -1,100 +1,53 @@
 const discord = require("discord.js")
+const {dispHelp} = require("./help");
+const {randInt, randFloat} = require("./randnbr");
+const {randList, randPeople} = require("./randtext");
 
 module.exports = {
     description: 'Bot gives a random number, between the specified bounds if they exist',
-    usage: 'random <type> <params>',
+    usage: 'random <type> [params]',
```
Comment (zaosoula) [inline commands/random/index.js]: *usage* doesn't need the command name

```suggestion
    usage: '<type> [params]',
```

File: commands/random/index.js [hunk 3]
```diff
@@ -1,100 +1,53 @@
 const discord = require("discord.js")
```
Comment (zaosoula) [inline commands/random/index.js]: Discord.JS isn't used in this file
```suggestion
```

**Review by zaosoula — CHANGES_REQUESTED** [review]

**Review by Fabio-web — APPROVED** [review]

**Review by Maxi-Mega — COMMENTED** [review]
Merged suggested changes

**Pull request merged**